zonemaster/zonemaster-engine
# Updates translation to Swedish (Engine)

## Purpose

This PR updates the translation (PO file) for Swedish.

## How to test this PR

Change to Swedish and verify that translation comes out as expected for updated msgids and msgstrs.
